src/FSharp.Control.TaskSeq.Test/TaskSeq.Realworld.fs

```diff
@@ -102,20 +100,9 @@ type ``Real world tests``(output: ITestOutputHelper) =
 
     [<Fact(Skip = "Broken test, faulty streaming test-implementation")>]
     let ``Reading a 10MB buffered IAsync stream from start to finish`` () = task {
-        let mutable count = 0
         use reader = AsyncBufferedReader(output, Array.init 2048 byte, 256)
         let expected = Array.init 256 byte
 
-        let ts = taskSeq {
-            for data in reader do
-                do count <- count + 1
-
-                if count > 40960 then
-                    failwith "Too far!!!!!!" // ensuring we don't end up in an endless loop
-
-                yield data
-        }
-
```

Comment:
@bartelink what was the reason for removing this? If it is not called, the better fix would be to call it. Also, it is marked `Skip`, so it is better to leave as-is, or fix the test ;).

Reply:
Yes, it's a draft, I was drawing attention to the fact that it was dead (there's a line removed elsewhere which probably had intent behind it the removal was not intended as final; just to convey that the compiler would not miss it)